src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -2564,8 +2568,15 @@ def openshift_create(cmd, client, resource_group_name, name,  # pylint: disable=
 
     try:
         # long_running_operation_timeout=300
-        return sdk_no_wait(no_wait, client.create_or_update,
-                           resource_group_name=resource_group_name, resource_name=name, parameters=osamc)
+        result = sdk_no_wait(no_wait, client.create_or_update,
+                             resource_group_name=resource_group_name, resource_name=name, parameters=osamc)
+        result = LongRunningOperation(cmd.cli_ctx)(result)
```

Comment:
We don't use `result` after this point in this method. Should it be `return`ed? Why are we making a `LongRunningOperation` out of it if it isn't used?